class CompoundException(Exception):
    def __init__(self):
        pass